data/hai/hai reveal 4 middle.txt

```diff
@@ -1508,27 +1508,115 @@ mission "Hai Reveal [A14] A Time of Trouble"
 			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
-			`	Teeneep takes a moment to respond, "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
-			`	The President replies, "The Minister for Foreign Affairs has been keeping Cabinet apprised of the major details and proposals reported back by Deputy Minister Yang. Under the circumstances we believe it would be appropriate for us to attempt a joint effort to address your concerns. We agree to making this effort against one target, and to reassessing the situation afterwards depending upon what is discovered in that process. The Republic representatives there with you now will work with you to identify that target and plan the action accordingly."`
+			`	Teeneep looks relieved by this suggestion.`
+			choice
+				`	"That sounds like a viable solution that will benefit all parties. How does the Hai representative respond?"`
+				`	"Teeneep, will that satisfy the Hai?"`
+					goto silence
+			`	Both human leaders nod in agreement with you.`
+			label silence
+			`	Teeneep takes a moment to respond. "That sounds like an acceptable future state... but I cannot sign any treaty yet. The Hai will not be satisfied until our kidnapped brethren are returned."`
+			`	The President replies, "The Minister for Foreign Affairs has been keeping the Cabinet apprised of the major details and proposals reported back by Deputy Minister Yang. Under the circumstances we believe it would be appropriate for us to attempt a joint effort to address your concerns. We agree to making this effort against one target, and to reassessing the situation afterwards depending upon what is discovered in that process. The Republic representatives there with you now will work with you to identify that target and plan the action accordingly."`
 			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
 			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
-			`	A number of people involved in the busywork of bureaucracy start slipping out of the room, doubtlessly to get a head start on certain preparations as Teeneep and the leaders on the screen conclude some formal thanks for continued cooperation and other matters of protocol. You are too visible to slip away though and have to wait.`
+			choice
+				`	"Thank you, President, Chancellor, and Hai representative, for a successful resolution."`
+				`	"All of us should remember this day: the day three nations came together to form a lasting peace."`
+					goto peace
+				`	"Well, I guess I saved the galaxy yet again. Captain <last>, signing out!"`
+					goto appalled
```

Comment:
No.

Reply:
Damn.

Reply:
Captain Last no longer signs out rudely. I left the other two closing statements.